<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QWidget" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MapTreeWidget" name="m_pMapTree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::MoveAction</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <property name="allColumnsShowFocus">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>75</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="MapTabWidget" name="m_pMapEditor">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>541</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>map tree:</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="m_pMappingModeTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_pTileMapping">
    <attribute name="title">
     <string>Tiles</string>
    </attribute>
    <widget class="QTabWidget" name="m_pTileTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>231</width>
       <height>171</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="TileView" name="m_pTiles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Tiles</string>
      </attribute>
     </widget>
     <widget class="TileSetPrototypeDragTreeWidget" name="m_pTileSets">
      <attribute name="title">
       <string>TileSets</string>
      </attribute>
     </widget>
     <widget class="AutoTileView" name="m_pAutoTiles">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>AutoTiles</string>
      </attribute>
     </widget>
    </widget>
    <widget class="MapBrushWidget" name="m_pLHBrush" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="MapBrushWidget" name="m_pRHBrush" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="m_pObjectMapping">
    <attribute name="title">
     <string>Objects</string>
    </attribute>
    <widget class="QTabWidget" name="m_pObjectTabs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>261</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="WorldObjectPrototypeDragTreeWidget" name="m_pWorldObjects">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>WorldObjects</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_pRevert">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>250</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pShowGrid">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>grid:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>zoom:</string>
   </property>
  </widget>
  <widget class="QSlider" name="m_pZoom">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="minimum">
    <number>30</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QFrame" name="m_pLayer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QSpinBox" name="m_pCurLayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="m_pMapTreeLabel_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>layer no.:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pLayerBackground">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>background</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pLayerForeground">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>foreground</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TileView</class>
   <extends>QWidget</extends>
   <header>tileview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>maptree.h</header>
  </customwidget>
  <customwidget>
   <class>MapTabWidget</class>
   <extends>QTabWidget</extends>
   <header>mapview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorldObjectPrototypeDragTreeWidget</class>
   <extends>QWidget</extends>
   <header>dragtreewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AutoTileView</class>
   <extends>QWidget</extends>
   <header>tileview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapBrushWidget</class>
   <extends>QWidget</extends>
   <header>mapbrushwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TileSetPrototypeDragTreeWidget</class>
   <extends>QWidget</extends>
   <header>dragtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
